--- a/setup.py
+++ b/setup.py
@@ -306,11 +306,11 @@
     basename = os.path.basename(os.path.dirname(__file__))
     basename_components = basename.split("-")
     if (
-        len(basename_components) == 3
+        len(basename_components) in (2, 3)
         and basename_components[0] == "bpython"
-        and basename_components[2] == "release"
+        and (len(basename_components) == 2 or basename_components[2] == "release")
     ):
         version = basename_components[1]
